### PR TITLE
[9.0][product_bundle] New module to have product bundles.

### PR DESCRIPTION
**Pull request by yajo**
This is a continuation from https://github.com/OCA/product-attribute/pull/169, dramatically simplified.

@Tecnativa

### CODE REVIEW

File: product_bundle/models/product.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from openerp import _, api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools.float_utils import float_round
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    bundle_ok = fields.Boolean(
```
Comment (carlosdauden) [inline product_bundle/models/product.py:12]: The name of this field could be "is_bundle"?

**Review by carlosdauden — COMMENTED** [review]
When you are creating a new bundle (product.template), if you don't save before than add Bundled products, Bundled products don't save.

File: product_bundle/models/product.py (hunk 1 above)
Comment (yajo) [reply product_bundle/models/product.py:12]: The `_ok` suffix seems to be the standard...

**Comment by rafaelbn** [comment]
Hi @Yajo , which is the status of this PR? Why is WIP @pedrobaeza ? Thanks!

cc @jjscarafia here you are ;-D

**Comment by yajo** [comment]
This is to be reviewed.

File: product_bundle/README.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+==============
+Product Bundle
+==============
+
+This module extends the functionality of products to support having a bundle of
+products in a single one.
+
+Internally, bundles will always be consumable products, but you will be able to
+see their stock, which is extracted from the maximum amount of stockable
```
Comment (rafaelbn) [inline product_bundle/README.rst:13]: s/maximum/minimum

**Review by rafaelbn — APPROVED** [review]
tested in runbot

**Comment by rafaelbn** [comment]
@pedrobaeza this is not a WIP is NR , please change tag, thanks. For me is 👍

**Comment by pedrobaeza** [comment]
Stock part is not yet covered.

File: product_bundle/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+# Translators:
+# Albert Cervera i Areny <albert@nan-tic.com>, 2009
+msgid ""
+msgstr ""
+"Project-Id-Version: odoo-addons (8.0)\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-09-09 23:57+0000\n"
+"PO-Revision-Date: 2015-09-10 00:00+0000\n"
+"Last-Translator: Juan Jose Scarafia <scarafia.juanjose@gmail.com>\n"
+"Language-Team: Spanish (http://www.transifex.com/adhoc/ingadhoc-odoo-addons-8-0/language/es/)\n"
```
Comment (jjscarafia) [inline product_bundle/i18n/es.po:14]: I believe that this module came, in some way, from product_pack (https://github.com/ingadhoc/product/tree/9.0/product_pack). What about authors?
Thanks!

Comment (pedrobaeza) [reply product_bundle/i18n/es.po:14]: No, that's only something not updated. We created from the scratch, but @Yajo reuse the folder and didn't remove these files. You only need to check code to see that nothing is equal...

Comment (jjscarafia) [reply product_bundle/i18n/es.po:14]: ok, just wondering, no need to check code. Thanks!

**Comment by pedrobaeza** [comment]
Closing for now as we don't have any project requiring this and there's still work to do. Anyone wanting to work on this can retake the work.

**Comment by rafaelbn** [comment]
It would be better to merge it, as it was working. Just a known issues about stock. But any way... here it is

**Comment by rafaelbn** [comment]
Working in https://github.com/OCA/product-attribute/pull/498

**Pull request closed without merging**